Provider/ImageProvider.php

```diff
@@ -174,6 +174,10 @@ public function generatePublicUrl(MediaInterface $media, $format)
      */
     public function generatePrivateUrl(MediaInterface $media, $format)
     {
+        if ($format == 'reference') {
```

Comment:
@phansys Forgot this, didn't see your link.

But, It's not on documentation nor on stable version of the fixer: https://github.com/FriendsOfPHP/PHP-CS-Fixer/find/v1.9

So I think we should ignore this before it's official (or not).